src/main/java/com/google/cloud/teleport/v2/bigquery-gcs/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,319 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can we not make this file at the parent pom level?

Reply:
Moved to parent pom level.